src/.npmignore

```diff
@@ -0,0 +1,2 @@
+.npmignore
```

Comment:
why have .npmignore here?

Reply:
force of habit.  I wasn't sure if `npm pack` would pull in `.npmignore` by default